src/p_mobj.h

```diff
@@ -416,7 +416,15 @@ extern int iquetail;
 // [FG] colored blood and gibs
 extern boolean colored_blood;
 
-#define PLAYER_SLOPE(a)	((((a)->lookdir / MLOOKUNIT) << FRACBITS) / 173) // angle to fixed approximation
+// angle to fixed_t approximation:
```

Comment:
Did I get this right? @ceski-1

Reply:
Why does the horizontal fov determine the vertical aiming?

Reply:
I guess it's actually just the only valid fov estimation that we have.

Reply:
> Did I get this right? @ceski-1

Looks good to me. Do you want to make `LOOKDIRMIN` and `LOOKDIRMAX` match the original autoaim range (-100 to 100)?

Reply:
> Why does the horizontal fov determine the vertical aiming?

Consider a side view. The slope of the line for calculating aiming is `z / y` where `z` is the vertical screen offset and `y` is `SCREENWIDTH / 2` or obfuscated as `160`. This is the distance of the line from the projection plane to the player's POV. Call it `viewdist`.

![side](https://github.com/fabiangreffrath/woof/assets/56656010/3ee45231-6d18-4e20-b858-e8ded4a49dc1)

Now look at the top view. Doom uses a horizontal FOV of 90 degrees which spans `SCREENWIDTH` or `320`. That gives a right triangle starting from the player's POV with a hypotenuse of `320`. The distance of the line from the projection plane to the player's POV is still `viewdist` and it bisects the triangle.

![top](https://github.com/fabiangreffrath/woof/assets/56656010/7db331de-e7d3-42c7-9c4e-9c535fc2e35b)

So that means:
```
tan(angle) = x / y
tan(angle) = opposite / adjacent
tan(angle) = 160 / viewdist
```
Where:
```
angle = FOV/2 = pi/4
tan(angle) = tan(pi/4) = 1
```
Therefore:
```
viewdist = SCREENWIDTH / 2
viewdist = 160
```

Reply:
> > Did I get this right? @ceski-1
> 
> Looks good to me. Do you want to make `LOOKDIRMIN` and `LOOKDIRMAX` match the original autoaim range (-100 to 100)?

The code is derived from the Heretic implementation which already had its bounds set to -110..90:
https://github.com/chocolate-doom/chocolate-doom/blob/c6cbb05ca638fd7d603860406eb5253d358b45e2/src/heretic/p_user.c#L241

BTW, native speakers, is it "spans across 320 px" or "spans over 320 px" or simply "spans 320 px"?

Reply:
@ceski-1 would it help with the directional audio implementation if the boundaries were -100..100?

Reply:
@fabiangreffrath It's not important. Currently, the auto-aim range [-100, 100] is different than direct aiming [-110, 90]. If that bugs you, adjust the range. Otherwise, it's nothing to worry about. Example at the upper limit:

https://github.com/fabiangreffrath/woof/assets/56656010/c208ac8f-0b43-400e-bb02-02d5a44074aa

Test wad: [looktest.zip](https://github.com/fabiangreffrath/woof/files/11860206/looktest.zip)

Reply:
> @fabiangreffrath It's not important. Currently, the auto-aim range [-100, 100] is different than direct aiming [-110, 90]. If that bugs you, adjust the range. Otherwise, it's nothing to worry about. 

Now I get it, thanks for the example. I'll have to sleep a night about it, but chances are high I'll adjust this. Now that we have turned our backs against the Heretic implementation anyway... 😉

Reply:
Fun fact: In Heretic this didn't seem to make a difference, because here autoaim us always attempted and direct aiming only applies if autoaim fails.